src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -100,7 +567,7 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren('c')).toEqual([span('c:2')]);
   });
 
-  it('works on roots with animation priority before deferred work', () => {
+  it('handles interleaved lo-pri and hi-pri work', () => {
```

Comment:
Oh sorry. You're right that it's not directly used, but for example, when I was implementing `scheduleWork`, I branched incorrectly because I thought `HighPriority` was supposed to be scheduled with `scheduleHighPriWork` due to the confusing naming.